data/glade/gecoscon.glade

```diff
@@ -82,7 +82,7 @@
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
                         <property name="xalign">0.69999998807907104</property>
-                        <property name="pixbuf">../media/logo-gecos.png</property>
+                        <property name="pixbuf">../media/btn-update.png</property>
```

Comment:
Replacing this GECOS logo with a btn-update.png seems like a mistake to me.

Other buttons are being replaced too. Is there a reason?